libs/financial-aid/shared/src/lib/const.ts

```diff
@@ -48,3 +60,10 @@ export const getNextPeriod = {
 }
 
 export const apiBasePath = 'api/financial-aid'
+
+export const scrollToId = (id: string) => {
```

Comment:
I know we should split better into to helper files, but does this belong in `utils` instead of const?